ansible/roles/contrail-ansible-deployer/templates/inventory/group_vars/container_hosts.yml.j2

```diff
@@ -9,7 +9,7 @@ contrail_configuration:
   LINUX_DISTR: centos7
   CONTAINER_REGISTRY: "{{ docker_registry }}"
   PHYSICAL_INTERFACE: "{{ physical_interface }}"
-  VROUTER_GATEWAY: 172.16.0.1
+  VROUTER_GATEWAY: "{{ ansible_default_ipv4.gateway }}"
```

Comment:
Does `ansible_default_ipv4.gateway` contain the IP address from management or data subnet?

Reply:
This is IP from management network. This value is used to set up default routing on controller. Our dev-ops team also seems to be using IP from management interface (they have fixed value 172.16.0.1) - here you can find facts from their controller http://logs.opencontrail.org/35/39935/1/check/contrail-systest-centos74-kolla-ocata/882f0f0/zuul-info/host-info.kolla-aio.yaml

Reply:
However it looks suspicious to me